gyp_xwalk

```diff
@@ -351,6 +351,9 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  if sys.platform == 'win32':
```

Comment:
@rakuco I did use <code>'foo%': 1</code> not <code>'foo%: 1'</code>
I paste my test as bellow which following your suggestion:
```
$ git diff
diff --git a/build/common.gypi b/build/common.gypi
index 6bf5877..b2d582d 100644
--- a/build/common.gypi
+++ b/build/common.gypi
@@ -26,8 +26,10 @@
       ['OS=="win"', {
         # Use RSSDK for video capture by default on Windows
         'use_rssdk%': 1,
+        'enable_widevine%': 1,
       }, {
         'use_rssdk%': 0,
+        'enable_widevine%': 0,
       }],
     ],
   },
diff --git a/gyp_xwalk b/gyp_xwalk
index a7a2e3f..c2dcc94 100755
--- a/gyp_xwalk
+++ b/gyp_xwalk
@@ -351,9 +351,6 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')

-  if sys.platform == 'win32':
-    args.append('-Denable_widevine=1')
-
   if not use_analyzer:
     print 'Updating projects from gyp files...'
     sys.stdout.flush()
```
Then,
```
python gyp_xwalk.py -Dmediacodecs_EULA=1
ninja -C ../out/Debug xwalk
../out/Debug/xwalk.exe --remote-debugging-port=9222 http://shaka-player-demo.appspot.com/?dash;asset=assets/car_segmenttemplate.mpd
```
The result is the test video can't be played. If you see the remote debug info, it shows that <code>DOMException: Unsupported keySystem</code>. I think It is because you don't really enable  enable_widevine=1. I know <code>widevinecdm</code> stub will be generated. That's because you redefined a flag 'enable_widevine' which overwrites the default one.

Is any step wrong here or I misunderstand your meaning?